welcome/index.adoc

```diff
@@ -277,29 +284,36 @@ You can xref:../storage/expanding-persistent-volumes.adoc#expanding-persistent-v
 
 ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 - **xref:../applications/quotas/quotas-setting-per-project.adoc#quotas-setting-per-project[Set resource quotas]**: Choose from CPU, memory, and other system resources to xref:../applications/quotas/quotas-setting-per-project.adoc#quotas-setting-per-project[set quotas].
-endif::[]
+endif::openshift-enterprise,openshift-webscale,openshift-origin[]
 
 - **xref:../applications/pruning-objects.adoc#pruning-objects[Prune and reclaim resources]**: Reclaim space by pruning unneeded Operators, groups, deployments, builds, images, registries, and cron jobs.
 
 - **xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-infrastructure-practices.adoc#scaling-cluster-monitoring-operator[Scale] and xref:../scalability_and_performance/using-node-tuning-operator.adoc#using-node-tuning-operator[tune] clusters**: Set cluster limits, tune nodes, scale cluster monitoring, and optimize networking, storage, and routes for your environment.
 
-- **Update a cluster**:
+- **xref:..//updating/understanding_updates/intro-to-updates.html[Update a cluster]**:
 Use the Cluster Version Operator (CVO) to upgrade your {product-title} cluster. If an update is available from the OpenShift Update Service (OSUS), you apply that cluster update from the {product-title} xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[web console] or the xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[OpenShift CLI] (`oc`).
 
-- **xref:../updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc#update-service-overview_updating-restricted-network-cluster-osus[Using the OpenShift Update Service in a disconnected environment]**: Learn about installing and managing a local OpenShift Update Service for recommending {product-title} updates in disconnected environments.
+- **xref:../updating/updating_a_cluster/updating_disconnected_cluster/index.html[Using the OpenShift Update Service in a disconnected environment]**: Learn about installing and managing a local OpenShift Update Service for recommending {product-title} updates in disconnected environments.
```

Comment:
Should this one also use the _You can install_ structure like most/all of the other links here?

Reply:
Should this one also use the _You can_ structure like most/all of the other links here? 

```suggestion
- **xref:../updating/updating_a_cluster/updating_disconnected_cluster/index.html[Using the OpenShift Update Service in a disconnected environment]**:  You can use the OpenShift Update Service for recommending {product-title} updates in disconnected environments.
```